tox-travis.ini

```diff
@@ -0,0 +1,220 @@
+# Tox (http://tox.testrun.org/) is a tool for running tests
+# in multiple virtualenvs. This configuration file will run the
+# test suite on all supported python versions. To use it, "pip install tox"
+# and then run "tox" from this directory.
+[tox]
+# tox.ini and tox-travis.ini are copies of the same content with different envlist commented out
+# tox-travis.ini will have pyXX-clean and all pyXX-installers in the default envlist, while the developer tox.ini 
+# does not have clean and only has one pyXX-installers
+# Uncomment this line for tox.ini
+# envlist = py38-build_test,py38-codegen,py38-installers,py{35,36,37,38,py3}-test,py38-flake8,py38-docs,py38-pkg
+# Uncomment this line for tox-travis.ini
+envlist = py38-clean,py38-build_test,py38-codegen,py{35,36,37,38,py3}-installers,py{35,36,37,38,py3}-test,py38-flake8,py38-docs,py38-pkg
+skip_missing_interpreters=True
+ignore_basepython_conflict=True
+skipsdist = true
+toxworkdir = .tox/{env:BITNESS:64}
+
+[testenv]
+description =
+    test: Run tests
+    build_test: Test the build scripts
+    clean: Clean code generated files
+    codegen: Run code generation step
+    installers: Build the installers (wheels and sdists)
+    flake8: Run static analysis
+    docs: Generate documentation
+    pkg: Verify the package
+    nidcpower_system_tests: Run system tests for nidcpower
+    nidigital_system_tests: Run system tests for nidigital
+    nidmm_system_tests: Run system tests for nidmm
+    nifgen_system_tests: Run system tests for nifgen
+    niscope_system_tests: Run system tests for niscope
+    niswitch_system_tests: Run system tests for niswitch
+    nimodinst_system_tests: Run system tests for nimodinst
+    nise_system_tests: Run system tests for nise
```

Comment:
Do you think it would make sense to codegen this file?
Maybe then we wouldn't have two copies of so much repetitive stuff.

(could be done in a separate PR if you agree, we just need to open an issue)

Reply:
I thought about that, and we could. #1202